assets/js/admin-external-connection.js

```diff
@@ -6,6 +6,8 @@ import { addQueryArgs, isURL, prependHTTP } from '@wordpress/url';
 import { speak } from '@wordpress/a11y';
 import compareVersions from 'compare-versions';
 
+const { __, sprintf } = wp.i18n;
+
```

Comment:
```suggestion
import { speak } from '@wordpress/a11y';
import { __, sprintf } from '@wordpress/i18n';
import compareVersions from 'compare-versions';

```

:1234: Use `import` elsewhere in the PR too.